mmv1/third_party/terraform/services/compute/resource_compute_router_bgp_peer_test.go.erb

```diff
@@ -206,6 +207,75 @@ func TestAccComputeRouterPeer_Ipv6Basic(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
+func TestAccComputeRouterPeer_Ipv4Basic(t *testing.T) {
```

Comment:
This test and `TestAccComputeRouterPeer_UpdateIpv4` can be combined into one test to test resource creating and updating.

Reply:
Done